pyramid/path.py

```diff
@@ -337,8 +337,13 @@ def _pkg_resources_style(self, value, package):
                 value = package.__name__
             else:
                 value = package.__name__ + value
-        return pkg_resources.EntryPoint.parse(
-            'x=%s' % value).load(False)
+        # Calling EntryPoint.load with an argument is deprecated.
+        # See https://pythonhosted.org/setuptools/history.html#id8
+        ep = pkg_resources.EntryPoint.parse('x=%s' % value)
+        if hasattr(ep, 'resolve'):
```

Comment:
Ugh...  But done in ae16a63.  Let me know if you'd rather I rebase and merge the two commits.